src/Symfony/Component/Ldap/Tests/Adapter/ExtLdap/EntryManagerTest.php

```diff
@@ -44,4 +44,37 @@ public function testGetResources()
         $entryManager = new EntryManager($connection);
         $entryManager->update($entry);
     }
+
+    /**
+     * @see https://tools.ietf.org/html/rfc4514#section-3
+     *
+     * @dataProvider moveWithRFC4514DistinguishedNameProvider
+     */
+    public function testMoveWithRFC4514DistinguishedName(string $dn, string $expectedRdn)
+    {
+        $connection = $this->createMock(Connection::class);
+
+        $entry = new Entry($dn);
+        $entryManager = new EntryManager($connection);
+
+        $method = (new \ReflectionClass(EntryManager::class))->getMethod('parseRdnFromEntry');
+        $method->setAccessible(true);
+
+        $cn = $method->invokeArgs($entryManager, [$entry, 'a']);
+
+        $this->assertSame($expectedRdn, $cn);
+    }
+
+    /**
+     * @return array<array{0: string, 1: string}>
```

Comment:
should be removed, we don't care about types in test cases

Reply:
Didn't know that

Reply:
Sure, I've removed the DocBlock